railties/lib/rails/generators/rails/app/templates/docker-entrypoint.tt

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Used by the Dockerfile to start the application with a prepared database
+
+# Create new or migrate existing database
+./bin/rails db:prepare
+
+# Start the server
+exec ./bin/rails server "$@"
```

Comment:
I think if that's what you want, you're probably better off overwriting the --entrypoint, since you don't want the db:prepare command to be run again.

Reply:
Open to suggestions. I'm keen to get the default path for "start this app in production" to Just Work, with no gotchas or whatever. I suppose it isn't the end of the world to run that db:prepare again? Or we could set the default, somehow, to be server, and only run it then, but allow it to be overwritten?